notes/20200217_meeting.md

```diff
@@ -34,11 +34,15 @@ Other benefits of RAP include:
 ### Improved quality: 
 - no manual steps (in the part of the process selected for RAP)
 - peer review of code
-- testing (*unclear how prescreptive we want to be here*)
+- testing (*unclear how prescriptive we want to be here*)
 
 ### Increased trust:
+- code and documentation open to anyone (if possible)
+- software open to anyone?
```

Comment:
Definitely agree that these two increas trust, but perhaps it's most important that the code/documentation/software are open to anyone (current and future) in a department? It would help increase trust by analysts and their managers, although not for users but maybe that's a big ask for an MVP?

Reply:
I think I'd lean towards having the code be open to anyone even for an MVP. It doesn't seem like a huge amount of 
extra work but I suppose that might depend on whether there's a suitable platform available for sharing code widely.

Reply:
If organisations want to be on the RAP train, then encouraging them to give their analysts a suitable platform sounds like a thing we should do.